libs/external-router/src/model/resolvable-route.ts

```diff
@@ -0,0 +1,17 @@
+import { DaffExternalRouteType } from './route-type';
+
+/**
+ * A `DaffResolvableRoute` describes the relationship between a "type" of route
+ * and a URI. E.g.
+ *
+ * ```ts
+ * export const resolvableRoute: DaffResolvableRoute = {
+ * 	url: "some/path/to-a-resource"
+ * 	type: "RESOURCE_TYPE"
+ * }
+ * ```
+ */
+export interface DaffResolvableRoute {
```

Comment:
What do you think of the name, `DaffResolvableUrl` instead? "Route" makes me think it is internal to angular. "Url" makes me think it is an external url.

Reply:
419261fb